superset-frontend/plugins/plugin-chart-echarts/package.json

```diff
@@ -25,13 +25,13 @@
   ],
   "dependencies": {
     "d3-array": "^1.2.0",
-    "echarts": "^5.4.1",
     "lodash": "^4.17.21",
     "moment": "^2.30.1"
   },
   "peerDependencies": {
     "@superset-ui/chart-controls": "*",
     "@superset-ui/core": "*",
+    "echarts": "*",
```

Comment:
I vaguely remember there having been a discussion about why this was needed as a core dependency. Is this really necessary, or can we leave `echarts` as a dependency only on the ECharts plugin?

Reply:
We use it in one of the new controls and thus also need it as a core dependency.